specification/applicationinsights/resource-manager/microsoft.insights/stable/2015-05-01/analyticsItems_API.json

```diff
@@ -0,0 +1,401 @@
+                    }
+                },
+                "x-ms-examples": {
+                    "AnalyticsItemDelete": {
+                        "$ref": "./examples/AnalyticsItemDelete.json"
+                    }
+                }    
+            }
+        }
+    },
+    "definitions": {
+        "ApplicationInsightsComponentAnalyticsItem": {
+            "description": "Properties that define an Analytics item that is associated to an Application Insights component.",
+            "properties": {
+                "Id": {
+                    "type": "string",
+                    "description": "Internally assigned unique id of the item definition."
+                },
+                "Name": {
+                    "type": "string",
+                    "description": "The user-defined name of the item."
+                },
+                "Content": {
+                    "type": "string",
+                    "description": "The content of this item"
+                },
+                "Version": {
+                    "type": "string",
+                    "description": "This instance's version of the data model. This can change as new features are added.",
+                    "readOnly": true
+                },
+                "Scope": {
+                    "description": "Enum indicating if this item definition is owned by a specific user or is shared between all users with access to the Application Insights component.",
+                    "type": "string",
+                    "enum": [
+                        "shared",
+                        "user"
+                    ],
+                    "x-ms-enum": {
+                        "name": "ItemScope",
+                        "modelAsString": false
+                    }
+                },
+                "Type": {
+                    "description": "Enum indicating the type of the Analytics item.",
+                    "type": "string",
+                    "enum": [
+                        "Query",
+                        "query",
+                        "Function",
+                        "function",
+                        "Folder",
+                        "folder",
+                        "Recent",
+                        "recent"
+                    ],
+                    "x-ms-enum": {
+                        "name": "ItemType",
+                        "modelAsString": false
+                    }
+                },
+                "TimeCreated": {
+                    "type": "string",
+                    "description": "Date and time in UTC when this item was created.",
+                    "readOnly": true
+                },
+                "TimeModified": {
+                    "type": "string",
+                    "description": "Date and time in UTC of the last modification that was made to this item.",
+                    "readOnly": true
+                },
+                "Properties": {
+                    "properties": {
```

Comment:
please pull this out into its own entry in `schemas` - that way you can control the name chosen during code generation (as is, `autorest` would pick some name which is usually horrible).

Also, add `type: object` to the schema 😉